src/Microsoft.FeatureManagement/ConfigurationFeatureDefinitionProvider.cs

```diff
@@ -27,7 +27,9 @@ public sealed class ConfigurationFeatureDefinitionProvider : IFeatureDefinitionP
         private readonly ConcurrentDictionary<string, FeatureDefinition> _definitions;
         private IDisposable _changeSubscription;
         private int _stale = 0;
-        private readonly bool _microsoftFeatureManagementSchemaEnabled;
+        private long _initialized = 0;
+        private bool _microsoftFeatureManagementSchemaEnabled;
```

Comment:
https://github.com/microsoft/FeatureManagement-Dotnet/pull/421#discussion_r1558782830

Now microsoft schema is the primary schema. I think the `AddFeatureManagement(IConfiguration)` method should support it. Then we cannot decide whether microsoft schema is enabled in the constructor because of the `rootConfigurationFallback`

Reply:
So you're saying it should be removed from the constructor right?

Reply:
Oh, sorry. I forget to remove it.